src/docs/get-started/codelab.md

```diff
@@ -326,40 +331,44 @@ a child inside the existing `MyApp` stateless widget.
  3. Add the `build()` method to `RandomWordsState`:
 
     <!-- skip -->
+    <?code-excerpt "3-stateful-widget/lib/main.dart (RandomWordsState)" indent-by="2" replace="/(\n  )(.*)/$1[[highlight]]$2[[\/highlight]]/g"?>
     {% prettify dart %}
       class RandomWordsState extends State<RandomWords> {
         [[highlight]]@override[[/highlight]]
         [[highlight]]Widget build(BuildContext context) {[[/highlight]]
-          [[highlight]]final wordPair = WordPair.random();[[/highlight]]
-          [[highlight]]return Text(wordPair.asPascalCase);[[/highlight]]
+        [[highlight]]  final wordPair = WordPair.random();[[/highlight]]
```

Comment:
I think Shams would prefer the indentation spaces to not be highlighted. Is that possible?

Reply:
This will be turned into a diff soon.

Reply:
Hmm, I can't use a diff against the code of the previous step because then it shows all of the class as new.

Personally, I find the aligned left margin highlight nicer (it seems to strengthen the idea that that "block" of code is what is being added.

Reply:
I don't have a strong feeling, but I'm pretty sure @sfshaza2 does.